### DISCUSSION TITLE
Multiple filter with CharFilter

### DISCUSSION THREAD

**Discussion by A-Levin** [carltongibson/django-filter#1505] [General]
I have model Task, and it has fk to Family model. 
`class ShortInspectionTaskFilter(filters.FilterSet):
    city = filters.CharFilter(field_name="family__city")
    class Meta:
        model = Task
        fields = ("city",)
`
how could I pass multiple values of cities to these filter?

**Comment by carltongibson** [comment]
Take a look at [ModelMultipleChoiceFilter](https://django-filter.readthedocs.io/en/stable/ref/filters.html#modelmultiplechoicefilter)
